site.yml

```diff
@@ -34,6 +34,9 @@
       when: serverspec.enabled|default("False")|bool
     - role: logging
       tags: ['common', 'logging']
+    - role: audit
+      tags: ['common', 'logging', 'audit']
+      when: audit.enabled|default('False')|bool
```

Comment:
I was thinking to let it be false by default because the audit logs grow pretty fast when it is enabled if you have a lot of rules.